README.md

```diff
@@ -5,29 +5,36 @@
 ### parse-and-store
 ```bash
 ./bblfsh-performance parse-and-store --help
-parse file(s) with golang benchmark output and store it in influx db
+parse file(s) with golang benchmark output and store it into a given storage
 
 Usage:
-  bblfsh-performance parse-and-store [--language=<language>] [--commit=<commit-id>] <file ...> [flags]
+  bblfsh-performance parse-and-store [--language=<language>] [--commit=<commit-id>] [--storage=<storage>] <file ...> [flags]
 
 Aliases:
   parse-and-store, pas, parse-and-dump
 
 Examples:
-WARNING! To access influx db corresponding environment variables should be set.
-Full example of usage script is the following:
+WARNING! To access storage corresponding environment variables should be set.
+Full examples of usage scripts are following:
 
+# for prometheus pushgateway
+export PROM_ADDRESS="localhost:9091"
+export PROM_JOB=pushgateway
+bblfsh-performance parse-and-store --language=go --commit=3d9682b --storage="prom" /var/log/bench0 /var/log/bench1
+
+# for influx db
 export INFLUX_ADDRESS="http://localhost:8086"
 export INFLUX_USERNAME=""
 export INFLUX_PASSWORD=""
 export INFLUX_DB=mydb
 export INFLUX_MEASUREMENT=benchmark
-bblfsh-performance parse-and-store --language=go --commit=3d9682b /var/log/bench0 /var/log/bench1
+bblfsh-performance parse-and-store --language=go --commit=3d9682b --storage="influxdb" /var/log/bench0 /var/log/bench1
```

Comment:
If you don't specify commit, I think we can take the `HEAD`

Reply:
I think your comment somehow relates to this thread https://reviewable.io/reviews/bblfsh/performance/1#-LjKkfkM8v5R3iKGtMKR
`bblfsh-performance` does not know/interact with git(yet) and primary aim is to run it in CI, anyways it can be done like `--commit=$(git rev-parse HEAD)`

Or are you talking about `HEAD` as a string?..

Reply:
just thought. You can also use `go-git` to resolve `HEAD` to concrete commit hash.

Reply:
but generally it's not super important (must have).

Reply:
filed issue on that https://github.com/bblfsh/performance/issues/7